MaxText/train.py

```diff
@@ -167,6 +167,11 @@ def train_step(model, config, state, data, dropout_rng):
   rng1, gen_aqt_rng = jax.random.split(dropout_rng)
   aqt_rng, rng2 = jax.random.split(gen_aqt_rng)
 
+  if config.per_device_batch_size < 1:
```

Comment:
Again do we need to branch here? Can't we always slice?

Reply:
Sure I removed it, it should be safe to always slice